app/src/main/java/com/fstyle/androidtrainning/service/ServicePlayMusic.java

```diff
@@ -0,0 +1,77 @@
+package com.fstyle.androidtrainning.service;
+
+import android.app.Service;
+import android.content.Intent;
+import android.media.MediaPlayer;
+import android.net.Uri;
+import android.os.IBinder;
+import android.provider.MediaStore;
```

Comment:
ERROR: Unused import: android.provider.MediaStore. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports